ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/helpers/CHelper.java

```diff
@@ -16,96 +16,51 @@
 package com.spectralogic.ds3autogen.c.helpers;
 
 
-import com.google.common.collect.ImmutableList;
-import com.spectralogic.ds3autogen.api.models.Ds3Element;
-import com.spectralogic.ds3autogen.api.models.Ds3EnumConstant;
 import com.spectralogic.ds3autogen.utils.Helper;
 
-import java.text.ParseException;
-import java.util.stream.Collectors;
-
-import static com.spectralogic.ds3autogen.utils.ConverterUtil.isEmpty;
 
 public class CHelper {
```

Comment:
java doesn't seem to support private classes..?

http://stackoverflow.com/questions/1913863/java-why-can-we-define-a-top-level-class-as-private